ghedesigner/ground_heat_exchangers.py

```diff
@@ -337,6 +408,22 @@ def simulate(self, method: TimestepType):
             self.loading = q_dot
 
             hp_eft, d_tb = self._simulate_detailed(q_dot, t, g)
+
+        elif method == TimestepType.DYNAMIC:
```

Comment:
As noted previously, this isn't a timestep method, so we may need to do some more work here to make sure things are in the right place.

Reply:
Done.